demisto_sdk/commands/validate/initializer.py

```diff
@@ -525,7 +531,10 @@ def git_paths_to_basecontent_set(
                 if obj:
                     obj.git_status = git_status
                     # Check if the file exists
-                    if git_status in (GitStatuses.MODIFIED, GitStatuses.RENAMED):
+                    if (
+                        git_status in (GitStatuses.MODIFIED, GitStatuses.RENAMED)
+                        or find_type_by_path(file_path) == FileType.METADATA
```

Comment:
```suggestion
                        or find_type_by_path(file_path) == FileType.METADATA and not gitstatus
```

git status can also be added files which may lead to issues with attempting to create old file object without an existing one.